guides/common/modules/con_smart-proxy-networking.adoc

```diff
@@ -0,0 +1,43 @@
+[id="{smart-proxy-context}-networking"]
+= {SmartProxy} networking
+
+The communication between {ProjectServer} and hosts registered to a {SmartProxyServer} is routed through that {SmartProxyServer}.
+{SmartProxyServer} also provides {Project} services to hosts.
+
+[NOTE]
+====
+Some outgoing traffic returns to {Project} to enable internal communication and security operations.
+====
+
+ifndef::satellite[]
+In a topology with hosts connecting to a {SmartProxyServer}, 
+endif::[]
+ifdef::satellite[]
+In xref:{project-context}-topology-with-hosts-connecting-to-a-{smart-proxy-context}[], 
+endif::[]
+{SmartProxy} provides a single endpoint for all host network communications so that in remote network segments, only firewall ports to the {SmartProxy} itself must be open.
+Hosts do not need direct access to the {ProjectServer} itself.
+
+// TODO: Replace graphic with simpler graphic and reference to "Port and firewall requirements"
+ifdef::satellite[]
+[id="{project-context}-topology-with-hosts-connecting-to-a-{smart-proxy-context}"]
+.{Project} topology with hosts connecting to a {SmartProxy}
+image::common/topology-isolated-satellite.png[{ProjectName} topology with a host]
+endif::[]
+
+ifndef::satellite[]
+In a topology with hosts connecting directly to {ProjectServer}, 
+endif::[]
+ifdef::satellite[]
+In xref:{project-context}-topology-with-hosts-connecting-directly-to-{project-context}-server[], 
+endif::[]
+hosts connect to {ProjectServer} rather than a {SmartProxy}.
+This applies also to {SmartProxies} themselves because the {SmartProxyServer} is a host of {ProjectServer}.
+Hosts need direct network access to {ProjectServer}.
```

Comment:
In general, hosts need access to _their_ Smart Proxy; which can be either Foreman Server or a Smart Proxy Server.

Reply:
Yes, but this is a description of networking requirements for the setup where you don't deploy a proxy. Which means that the sentence is valid.